plugins/catalog-backend/src/next/database/DefaultProcessingDatabase.ts

```diff
@@ -508,6 +518,61 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
     };
   }
 
+  async listAncestors(
+    txOpaque: Transaction,
+    options: ListAncestorsOptions,
+  ): Promise<ListAncestorsResult> {
+    const tx = txOpaque as Knex.Transaction;
+    const { entityRef } = options;
+    const entityRefs = new Array<string>();
+
+    let currentRef = entityRef;
```

Comment:
```suggestion
    let currentRef = entityRef.toLocaleLowerCase('en-US');
```

since this is untrusted user input being matched with a strict machine generated value